packages/client-sdk-nodejs/src/config/storage-configuration.ts

```diff
@@ -0,0 +1,171 @@
+import {MomentoLoggerFactory} from '../';
+import {Middleware} from './middleware/middleware';
+import {StorageTransportStrategy} from './transport/storage';
+
+/**
+ * Configuration options for Momento StorageClient
+ *
+ * @export
+ * @interface StorageConfiguration
+ */
+export interface StorageConfiguration {
```

Comment:
please remove everything except the bare minimum stuff from this interface for now... we haven't implemented middleware yet, we haven't implemented throwOnErrors... I think logger and transport are okay

Reply:
i apologize, it looks like y'all had already wired up the num connections and throwOnErrors stuff and I didn't realize it.

if you've already removed the config for these that's great - don't worry about undoing the actual implementations though.

I definitely do not want to ship the middleware stuff for this any time soon, i think we might try to redo that part. but the numconnections and throwOnErrors will likely come in handy soon. apologies if you did any unnecessary work to remove implementation stuff related to those, but if you haven't, you don't need to

Reply:
I took it out to be consistent with other sdks. Let's add the stuff back when required.